content/docs/forms.md

```diff
@@ -21,15 +21,15 @@ HTML form elements work a little bit differently from other DOM elements in Reac
 </form>
 ```
 
-This form has the default HTML form behavior of browsing to a new page when the user submits the form. If you want this behavior in React, it just works. But in most cases, it's convenient to have a JavaScript function that handles the submission of the form and has access to the data that the user entered into the form. The standard way to achieve this is with a technique called "controlled components".
+這個表格有 HTML 表格的預設行為，也就是在使用者遞交一個表格的時候去瀏覽一個新頁面。如果你想要在 React 中也有這樣的行為的話，直接用 HTML 是可行的。但是在大多數的情況中，有一個 JavaScript function 來處理遞交表格的功能並讀取使用者在表格中填入的資料是很方便的。要做到這樣，標準的方法是使用「controlled component」：
```

Comment:
The term `controlled` in others reviewing, I suggest it can be translated to `被控制的`, or maybe we may keep original term?

```suggestion
這個表格有 HTML 表格的預設行為，也就是在使用者提交一個表格的時候去瀏覽一個新頁面。如果你想要在 React 中也有這樣的行為的話，直接用 HTML 是可行的。但是在大多數的情況中，有一個 JavaScript function 來處理提交表格的功能並讀取使用者在表格中填入的資料是很方便的。要做到這樣，標準的方法是使用「controlled component」：
```